DCWL-2072  Amend README.md & build.sbt (#231)
hmrc/customs-declarations-information: README.md

# Customs Declarations Information

> The Customs Declarations Information allows users tobe able to request the status and versions of a declaration.

Reviewer: ```suggestion
The Customs Declarations Information API can be used to request the status and versions of a declaration.
```
